packages/basti/src/cli/commands/common/prompt-for-aws-target.ts

```diff
@@ -76,31 +95,53 @@ async function getTargets(): Promise<{
     'DB clusters',
     subCli
   );
-  const elasticacheClusters = await getTargetResources(
+  const elasticacheRedisClusters = await getTargetResources(
     async () => await getReplicationGroupsByClusterMode(),
-    'Redis clusters',
+    'Elasticache redis clusters',
     subCli
   );
-
-  const elasticacheNodes = await getTargetResources(
+  const elasticacheRedisNodes = await getTargetResources(
     async () => await getCacheClusters(),
-    'Redis nodes',
+    'Elasticache redis nodes',
```

Comment:
Please, uppercase the word Redis (here and in all the other user-facing messages) since it's a proper name.